statsmodels/tsa/holtwinters.py

```diff
@@ -732,7 +732,10 @@ def _predict(self, h=None, smoothing_level=None, smoothing_slope=None,
         # (s0 + gamma) + (b0 + beta) + (l0 + alpha) + phi
         k = m * seasoning + 2 * trending + 2 + 1 * damped
         aic = self.nobs * np.log(sse / self.nobs) + (k) * 2
-        aicc = aic + (2 * (k + 2) * (k + 3)) / (self.nobs - k - 3)
+        try: 
+            aicc = aic + (2 * (k + 2) * (k + 3)) / (self.nobs - k - 3)
+        except ZeroDivisionError: 
```

Comment:
Better to check that self.nobs - k - 3 > 0 since will get wrong result if self.nobs - k - 3 is negative.

Reply:
Hi,
I tried to edit holtwinter.py. Unfortunately, I can't change it... do you know how to fix it ?